src/EnergyPlus/SizingManager.cc

```diff
@@ -2461,29 +2472,11 @@ void ProcessInputOARequirements(EnergyPlusData &state,
     // DERIVED TYPE DEFINITIONS
     // na
 
+    auto &thisOARequirements(state.dataSize->OARequirements(OAIndex).OAFlowMethod);
+
     if (NumAlphas > 1) {
-        if (UtilityRoutines::SameString(Alphas(2), "Flow/Person")) {
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = OAFlowCalcMethod::PerPerson;
-        } else if (UtilityRoutines::SameString(Alphas(2), "Flow/Zone")) {
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = OAFlowCalcMethod::PerZone;
-        } else if (UtilityRoutines::SameString(Alphas(2), "Flow/Area")) {
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = OAFlowCalcMethod::PerArea;
-        } else if (UtilityRoutines::SameString(Alphas(2), "AirChanges/Hour")) {
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = OAFlowCalcMethod::ACH;
-        } else if (UtilityRoutines::SameString(Alphas(2), "Sum")) {
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = OAFlowCalcMethod::Sum;
-        } else if (UtilityRoutines::SameString(Alphas(2), "Maximum")) {
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = OAFlowCalcMethod::Max;
-        } else if (UtilityRoutines::SameString(Alphas(2),
-                                               "INDOORAIRQUALITYPROCEDURE")) { // Indoor Air Quality Procedure based on ASHRAE Standard 62.1-2007
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = DataSizing::OAFlowCalcMethod::IAQProcedure;
-        } else if (UtilityRoutines::SameString(
-                       Alphas(2), "PROPORTIONALCONTROLBASEDONOCCUPANCYSCHEDULE")) { // Proportional Control based on ASHRAE Standard 62.1-2004
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = DataSizing::OAFlowCalcMethod::PCOccSch;
-        } else if (UtilityRoutines::SameString(
-                       Alphas(2), "PROPORTIONALCONTROLBASEDONDESIGNOCCUPANCY")) { // Proportional Control based on ASHRAE Standard 62.1-2004
-            state.dataSize->OARequirements(OAIndex).OAFlowMethod = DataSizing::OAFlowCalcMethod::PCDesOcc;
-        } else {
+        thisOARequirements = static_cast<OAFlowCalcMethod>(getEnumerationValue(OAFlowCalcMethodNamesUC, UtilityRoutines::MakeUPPERCase(Alphas(2))));
```

Comment:
Ummm … shouldn’t this be thisOARequirements.OAMethod =?

Reply:
Oops, thanks so much for catching these!